test/adjoint/test_demos.py

```diff
@@ -36,6 +36,10 @@
     "solid_body_rotation.py": {
         r"solutions\.export\((.*?)\)": "",
     },
+    "burgers-goal_oriented.py": {
+        """"maxiter": 35""": """"maxiter": 3""",
```

Comment:
```suggestion
        """"maxiter": 35""": """"maxiter": 1""",
```
Thanks @acse-ej321! It's still quite long at 90 sec. Can we reduce it to just 1 iteration? We could do the same with point_discharge2d-goal_oriented and burgers-hessian, which are the second 2 longest at 66 and 51 seconds.

Reply:
@ddundo - how about 2 iterations? If the demos serve partially as integration tests it would be good to take it through at least one complete iteration including adapt and re-run.

Reply:
Looks good! 60sec is much better now :) thanks!